src/TWUtils.cpp

```diff
@@ -597,6 +597,10 @@ void TWUtils::updateRecentFileActions(QObject *parent, QList<QAction*> &actions,
 	}
 
 	for (int i = 0; i < numRecentFiles; ++i) {
+#if defined(Q_OS_WIN)
```

Comment:
The & mnemonic is used on all platforms, so please remove the `#if` clause

Reply:
done